docs/contracts/overview.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Smart Contract Overview
+sidebar_position: 1
+---
+
+# Contracts
```

Comment:
Why not keeping same title as side bar?

`Smart Contract Overview` vs `Contracts`.